### PR TITLE
network changes for new localhost api, still needs fixing

### PR DESCRIPTION
**Pull request by mattbarker016**
I added the new Network base URL. I also removed the old BusStop functions throughout the project. 

Currently, while listening on 3000, the `parseAllStops` function receives a JSON that it isn't expecting. The search bar will crash on tapping because the proper data wasn't received from the API call / put into the JSON. I changed `json.array!` to `json.arrayValue` to allow this for now, but this is where things go screwy.

**Network.swift, line 22-24**
````    
required init(json: JSON) throws {
    allStops = parseAllStops(json: json.array!)
}
````

screen shot 2017-08-28 at 3 09 40 pm

### CODE REVIEW

**Comment by AAAstorga** [comment]
I fixed the parseAllStops function

File: TCAT/Utilities/Network.swift [hunk 1]
```diff
@@ -9,27 +9,30 @@ import Foundation
...
             let name = stop["name"].stringValue
-            let location = stop["location"].arrayObject as! [Double]
-            let lat = location[0]
-            let long = location[1]
+            let location = stop["location"]
+            let lat = location["latitude"].doubleValue
+            let long = location["latitude"].doubleValue
             let busStop = BusStop(name: name, lat: lat, long: long)
             allStopsArray.append(busStop)
         }
```
Comment (dantheli) [inline TCAT/Utilities/Network.swift]: Could be nicer in a map

**Review by dantheli — APPROVED** [review]
Looks good. Was there a specific reason why you updated the platform in podfile to 9.3?

**Comment by mattbarker016** [comment]
@dantheli I added a default OS to shut up a Cocoapods warning.

**Comment by mattbarker016** [comment]
Alright gang, here's the latest and greatest rebased and ready to roll. I had to do quite a bit of rebasing, so @AAAstorga and @myo3 **please** make sure I kept the correct commits

**Pull request merged**